inst/extdata/energy/A26.globaltech_cost.csv

```diff
@@ -0,0 +1,17 @@
+# File: A26.globaltech_cost.csv
+# Title: Final energy delivery cost adders
+# Units: ?
```

Comment:
Please check with @pkyle or @mwisepnnl or someone about this.

Reply:
In general, these are the energy delivery cost adders, or the difference between producer prices and delivered energy prices to different consumer groups. The unit is 1975$/GJ, and the source for most of the items is the pre-2006 MiniCAM model inputs. Values of zero indicate that delivery costs are included elsewhere in the chain.
A26 also includes some grid reliability (backup) related costs. The backup_electricity cost represents the variable O&M cost of a gas turbine (levelized capital and fixed O&M costs are accounted elsewhere). No such cost is included here for csp_backup, because the turbine costs are already accounted in the primary electric technology costs.

Reply:
Thanks, @pkyle!